src/cmd/singularity/cli/build.go

```diff
@@ -273,6 +273,8 @@ func makeBundle(def build.Definition) *build.Bundle {
 		cp = &build.DebootstrapConveyorPacker{}
 	case "arch":
 		cp = &build.ArchConveyorPacker{}
+	case "localimage":
+		cp = &build.LocalConveyorPacker{}
```

Comment:
With 2.5.2, building with this def file:

```
Bootstrap: local
From: /tmp/image
```

Gives:

```
$ sudo singularity build /tmp/image2 recipe 
Using container recipe deffile: recipe
Sanitizing environment
Adding base Singularity environment to container
ERROR: 'Bootstrap' type not supported: local
Cleaning up...
```

I think the case at line 266 is actually a typo... and we can just switch `local` to `localimage`.

Reply:
Agreed that option should have just been `localimage` in the first place. Make that change as well as a quick change on how definitions for local image builds are generated when building from the source file itself to make it cleaner, imo